dockstore-common/src/main/scala/io/dockstore/common/Bridge.scala

```diff
@@ -86,6 +86,50 @@ class Bridge(basePath : String) {
     }
   }
 
+  /**
+    * Will throw an error if the file is an invalid workflow
+    * @param file
+    * @throws wdl4s.parser.WdlParser.SyntaxError
+    * @throws java.lang.NullPointerException
+    */
+  @throws(classOf[WdlParser.SyntaxError])
+  @throws(classOf[NullPointerException])
```

Comment:
Hmmm, this seems controversial. (i.e. I see this but other arguments too)
https://rules.sonarsource.com/java/tag/error-handling/RSPEC-1695

Sidestepping, is there a more appropriate exception, as in is there a specific condition where this throws NullPointerException?

Reply:
If I remember correctly the Null Pointer Exception catches the case where an import is missing. I can look a bit deeper to see if there is a better exception.